testsuite/features/profiles/github_runner/Docker/add_packages.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+cp /root/avahi-daemon.conf /etc/avahi/avahi-daemon.conf
+/usr/sbin/avahi-daemon -D
```

Comment:
Do we need, or it could even work, the avahi communication in between our containers here in the GH runner?

Reply:
Here avahi is needed for **docker** network at our gh runner.  However, our containers work at the **podman** network at our gh runner.